Compilation error with Vaadin 23.3.0

```
 ERROR in frontend/generated/jar-resources/fc-applayout/fc-fusion-layout.ts:27:31
 TS2307: Cannot find module '@hilla/frontend' or its corresponding type declarations.
     25 | import * as menuItemEndpoint from '@vaadin/flow-frontend/generated/MenuEndpoint';
nt/MenuItemDto';
   > 27 | import { EndpointError } from '@hilla/frontend';
        |                               ^^^^^^^^^^^^^^^^^
     28 | import { Router } from '@vaadin/router';
     29 |
     30 | @customElement('fc-fusion-layout')
```

--- a/src/main/resources/META-INF/resources/frontend/fc-applayout/fc-fusion-layout.ts
+++ b/src/main/resources/META-INF/resources/frontend/fc-applayout/fc-fusion-layout.ts
@@ -22,8 +22,8 @@ import { css, customElement, html, LitElement, state, property, query } from 'li
 import "@flowingcode/fc-applayout";
 import {FcAppLayoutElement} from "@flowingcode/fc-applayout/src/fc-applayout";
 import "@flowingcode/fc-menuitem";
-import * as menuItemEndpoint from '@vaadin/flow-frontend/generated/MenuEndpoint';
-import MenuItemDto from '@vaadin/flow-frontend/generated/com/flowingcode/addons/applayout/endpoint/MenuItemDto';
+import * as menuItemEndpoint from 'Frontend/generated/MenuEndpoint';
+import MenuItemDto from 'Frontend/generated/com/flowingcode/addons/applayout/endpoint/MenuItemDto';
 import { EndpointError } from '@hilla/frontend';
 import { Router } from '@vaadin/router';
 

